src/acceptance/app/dynamic_policy_test.go

```diff
@@ -148,4 +149,56 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 		})
 
 	})
+
+	Context("when scaling by responsetime", func() {
+
+		JustBeforeEach(func() {
+			bindService := cf.Cf("bind-service", appName, instanceName, "-c", policy).Wait(cfg.DefaultTimeoutDuration())
+			Expect(bindService).To(Exit(0), "failed binding service to app with a policy ")
+		})
+
+		AfterEach(func() {
+			unbindService := cf.Cf("unbind-service", appName, instanceName).Wait(cfg.DefaultTimeoutDuration())
+			Expect(unbindService).To(Exit(0), "failed unbinding service from app")
+		})
+
+		Context("when responsetime is greater than scaling out threshold", func() {
+			BeforeEach(func() {
+				policy = generateDynamicScaleOutPolicy(1, 2, "responsetime", 5000)
+				initialInstanceCount = 1
+			})
+
+			It("should scale out", func() {
+				finishTime := time.Duration(interval*2)*time.Second + 3*time.Minute
+				timeout := 2 * time.Minute
+				for i := 0; i < 10; i++ {
```

Comment:
why you run 10 times?  I guess a better way is to start a routine to continuously drive load before test and stop it after the test